loadtest/config.go

```diff
@@ -88,6 +88,8 @@ type InstanceConfiguration struct {
 	NumPosts int64 `default:"0" validate:"range:[0,]"`
 	// The target number of reactions to be created.
 	NumReactions int64 `default:"0" validate:"range:[0,]"`
+	// The target number of admin users to be created.
+	NumAdmins int64 `default:"10" validate:"range:[0,]"`
```

Comment:
This should probably be the same value we set in the `sample.json` file. So I'd say 0 should be the default.